README-app.md.jinja

```diff
@@ -2,7 +2,7 @@
 
 @{{ app_name }} is deployed in the following environments:
 {% for e in matrix_envs %}
-- @{{ e["name"] }} -> @{{ e["app_subdomain"] }}.@{{ e["zone_name"] }}
+- @{{ e["name"] }} -> `{% if app_url_type == "subdomain" %}@{{ e["app_subdomain"] }}.{% endif %}@{{ e["zone_name"] }}{% if app_url_type == "path" %}/<your-app-path>{% endif %}`
```

Comment:
for path based we do not know the path of the app yet, but there is no subdomain